### DISCUSSION TITLE
[IDEA] AdBlockID Annoyance

### DISCUSSION THREAD

**Discussion by realodix** [realodix/AdBlockID#758] [Ideas]
> misal dibuat baru filter utk annoyance gmn bro?
> 
> Cookie banner
> Video autoplay
> Pop up
> Dll
> 

_Originally posted by @3xploiton3 in https://github.com/realodix/AdBlockID/issues/702#issuecomment-771776457_

**Comment by realodix** [comment]
Bisa saja, namun Saya belum ada waktu untuk membuat AdBlockID Annoyance seperti Fanboy’s Annoyance dan AdGuard Annoyances.

Saya membuka kesempatan jika ada yang mau bantu kirim PR atau mengajukan diri sebagai maintainer untuk membuat dan merawat AdBlockID Annoyance.

**Comment by Jokopentil** [comment]
Aduh bro.....websit di indonesia banyak
Pengguna internet banyak
Iklan juga banyak jenis,
Pengguna filter kita juga banyak , dari malay brunai juga ada
Saya rasa perlu juga apa yg di usulkan
Mengingat jumlah pengguna internet🤔🤔🤔
